src/main/java/teammates/common/datatransfer/questions/FeedbackMcqQuestionDetails.java

```diff
@@ -525,4 +473,36 @@ public String getQuestionTypeChoiceOption() {
     public String validateGiverRecipientVisibility(FeedbackQuestionAttributes feedbackQuestionAttributes) {
         return "";
     }
+
+    private Map<String, Integer> collateAnsweringFrequencyForMcqOptions(List<FeedbackResponseAttributes> responses) {
```

Comment:
`collateAnswerFrequency` should do as the method name, no need for a too long one.